src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -345,15 +350,15 @@ If you want to keep your source maps secret and choose not to upload your source
   content=__alert_content
 %}
 
-## Context
+## Adding Context Data
 You can also set context when manually triggering events.
 
 ### Setting Context {#context}
 Sentry supports additional context with events. Often this context is shared among any issue captured in its lifecycle, and includes the following components:
 
 **Structured Contexts**
 
-: Specific structured contexts --- OS info, runtime information, etc.  This is typically set automatically.
+Specific structured contexts --- OS info, runtime information, etc.  This is typically set automatically.
 
```

Comment:
does this section include snippets or is it just a list of the kind of data? if its just a list it seems like a lot of filler. if its examples I wonder if there's a way to simplify it.

Reply:
I'm wondering if we can put this information in a table or something. I think it's important to document all the types of data a dev can collect. But I also agree the way the information is presented feels like it needs a trim or a way to condense it. Earlier today, @lobsterkatie and I were discussing the table solution.